### PR TITLE
spdk: disable loopback interface check on SPDK socket create

### PR DESCRIPTION
**Pull request by dsavitskiy**
Signed-off-by: Dmitry Savitskiy <dmitry.savitskiy@datacore.com>

### CODE REVIEW

**Comment by dsavitskiy** [comment]
bors try

File: io-engine/src/bdev/nvmx/channel.rs [hunk 1]
```diff
@@ -361,7 +361,7 @@ impl IoQpair {
 
         // Drop context object transformed previously into a raw pointer.
         unsafe {
-            Box::from_raw(connect_arg);
+            drop(Box::from_raw(connect_arg));
```
Comment (tiagolobocastro) [inline io-engine/src/bdev/nvmx/channel.rs:364]: presumably this is not related to the spdk changes?

**Review by tiagolobocastro — APPROVED** [review]

File: io-engine/src/bdev/nvmx/channel.rs (hunk 1 above)
Comment (dsavitskiy) [reply io-engine/src/bdev/nvmx/channel.rs:364]: It is not. Compiler prints warning on this.

**Review by mtzaurus — APPROVED** [review]

**Comment by dsavitskiy** [comment]
bors merge
